README.md

```diff
@@ -25,7 +25,7 @@ already.
 The syntax is as follows:
 
 ```hcl
-provider "tmc" {
+provider "tanzu-mission-control" {
```

Comment:
Update it according the registry.terraform.io

Reply:
Done